assemblies/client/pom.xml

```diff
@@ -200,6 +200,12 @@
                   <version>${pentaho-hadoop-shims-emr521.version}</version>
                   <type>kar</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.pentaho</groupId>
+                  <artifactId>pentaho-hadoop-shims-mapr60-kar</artifactId>
+                  <version>${pentaho-hadoop-shims-mapr60.version}</version>
```

Comment:
Move the version to the parent pentaho-karaf-assembly pom.xml in the dependencyManagement section.